.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "4.3"
```

Comment:
Should we support "5" as well? What about 0.12, since IIRC @arvind said he was behind a version or two?